src/ripple/app/misc/Manifest.h

```diff
@@ -167,18 +167,11 @@ struct ValidatorToken
 {
     std::string manifest;
     SecretKey validationSecret;
-
-private:
-    ValidatorToken(std::string const& m, SecretKey const& valSecret);
-
-public:
-    ValidatorToken(ValidatorToken const&) = delete;
```

Comment:
It seems like protection against certain methods of construction (default, and from a reference) is now absent. Is this intended?

Reply:
Yeah. The type is now basically a glorified `std::pair` and can use uniform initialization. I'd have used `std::pair` but I think the consensus of everyone on our team is that `.manifest` and `.validationSecret` are more helpful that `.first` and `.second`, which is a fair point.